python/triton/runtime/jit.py

```diff
@@ -349,12 +360,29 @@ def __setattr__(self, name, value):
         #   to be reinitialized
         if name == 'src':
             self.hash = None
-            JITFunction.cache_key.fget.cache_clear()
 
     def __repr__(self):
         return f"JITFunction({self.module}:{self.fn.__name__})"
 
 
+class MockTensor:
```

Comment:
I think we already have https://github.com/openai/triton/blob/master/python/triton/utils.py#L22 used by `reinterpret`. Could we move MockTensor there? Probably in the future `TensorWrapper` and `MockTensor` can be merged, but that would require refactoring some other files too.

Reply:
Done